components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -104,10 +107,57 @@ function Main() {
     }
   }
 
+  let inputFocused: boolean = false
+  let viewPortWithoutKeyboard = 0
+  let keyboardSize = 0
+
+  window.addEventListener('resize', () => {
+    if (!context.isMobile || !inputFocused || window.visualViewport == null) {
+      return
+    }
+    let viewPortWithKeyboard = window.visualViewport.height
```

Comment:
nit:
```suggestion
    const viewPortWithKeyboard = window.visualViewport.height
```

here and other places we don't modify after assigning